packages/synapse-react-client/src/SWC.index.ts

```diff
@@ -127,88 +47,166 @@ const SynapseContext = {
 }
 
 const SynapseComponents = {
-  Login: StandaloneLoginForm,
-  EntityForm,
-  UserCard,
-  StatisticsPlot,
-  HasAccess,
-  EvaluationCard,
-  EvaluationEditorPage,
-  AccessTokenPage,
-  AccountLevelBadges,
-  TermsAndConditions,
-  PageProgress,
-  SynapseHomepageV2,
-  SynapsePlansPage,
-  SynapseFooter,
-  EntityFinder,
-  ErrorPage,
-  EntityBadgeIcons,
-  DownloadCartPage,
-  ShowDownloadV2,
-  DownloadConfirmation: FolderDownloadConfirmation,
-  FullWidthAlert: FullWidthAlert,
-  SchemaDrivenAnnotationEditor,
-  SynapseNavDrawer,
-  FavoritesPage,
-  EntityCitation,
-  EntityModal,
   SynapseToastContainer,
   displayToast,
-  IconSvg,
-  UserProfileLinks,
-  DatasetItemsEditor,
-  EntityTypeIcon,
-  HelpPopover,
-  ProgrammaticTableDownload,
-  DirectProgrammaticDownload,
-  ProgrammaticInstructionsModal,
-  SkeletonButton,
-  QueryWrapperPlotNav,
-  StandaloneQueryWrapper,
-  ChangePassword,
-  ForumSearch,
-  DiscussionThread,
-  ReviewerDashboard,
-  ProvenanceGraph,
-  TrashCanList,
-  OAuthManagement,
-  CertificationQuiz,
-  HtmlPreview,
-  IDUReport,
-  EntityPageBreadcrumbs,
-  EntityActionMenu,
-  EntityPageTitleBar,
-  CreatedByModifiedBy,
-  TwoFactorAuthSettingsPanel,
-  TwoFactorBackupCodes,
-  TwoFactorEnrollmentForm,
-  SubscriptionPage,
-  OrientationBanner,
-  AvailableEvaluationQueueList,
-  AccessRequirementList,
-  TableColumnSchemaForm,
-  EntityHeaderTable,
-  AccessRequirementRelatedProjectsList,
-  CreateTableViewWizard,
-  TableColumnSchemaEditor,
-  SqlDefinedTableEditorModal,
-  EntityViewScopeEditorModal,
-  SubmissionViewScopeEditorModal,
-  AccessRequirementAclEditor,
-  CreateOrUpdateAccessRequirementWizard,
-  GoogleAnalytics,
-  CookiesNotification,
   getCurrentCookiePreferences,
-  EntityAclEditorModal,
-  SynapseChat,
-  RejectProfileValidationRequestModal,
-  GovernanceMarkdownGithub,
-  MarkdownGithubLatestTag,
-  ProjectDataAvailability,
-  EntityUploadModal,
-  CreateOrUpdateDoiModal,
-  UserAccessRequestHistoryPlace,
+  EntityForm: lazy(() => import('./components/EntityForm/EntityForm')),
```

Comment:
Rewrote all components as `lazy` - Vite will code-split, so the initial JS to download will be much smaller, depending on the page you're on (no more loading the Homepage assets when you never navigate to it!)